targets/disc-configs/lenovo-x1-carbon-disk-config.nix

```diff
@@ -0,0 +1,166 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+_: {
+  disko.devices = {
+    disk.disk1 = {
+      device = "DRIVE_PATH";
+      type = "disk";
+      content = {
+        type = "gpt";
+        partitions = {
+          boot = {
+            name = "boot";
+            size = "1M";
+            type = "EF02";
+          };
+          esp_a = {
+            name = "ESP_A";
+            size = "500M";
+            type = "EF00";
+            content = {
+              type = "filesystem";
+              format = "vfat";
+              mountpoint = "/boot";
+              mountOptions = ["umask=0077"];
+            };
+          };
+          esp_b = {
+            name = "ESP_B";
+            size = "500M";
+            type = "EF00";
+            content = {
+              type = "filesystem";
+              format = "vfat";
+              mountOptions = ["umask=0077"];
+            };
+          };
+          # LVM pool that is going to be encrypted
+          other_1 = {
+            name = "lvm_pv_1";
+            size = "250G";
+            content = {
+              type = "lvm_pv";
+              vg = "pool";
+            };
+          };
+          # LVM pool that is going to be passed to the Storage VM
+          other_2 = {
```

Comment:
Are VMs not encrypted?

Reply:
StorageVM partition is not going to be encrypted, since it is supposed that it stores encrypted data. In this case it allows us to backup and recover the stored data safely if needed.